care/facility/api/viewsets/notification.py

```diff
@@ -52,10 +52,10 @@ class DummyNotificationSerializer(Serializer):  # Dummy for Spec
     @action(detail=False, methods=["POST"])
     def notify(self, request, *args, **kwargs):
         user = request.user
-        if "facility" not in request.data:
+        if "facility" not in request.data or request.data["facility"] == "":
```

Comment:
Need more solid validation to check if the value is a UUID, but we'll build it later on.